src/types/ELMTypes.ts

```diff
@@ -167,6 +169,8 @@ export interface ELMExpression {
   localId?: string;
   /** Locator in the original CQL file. Only exists if compiled with this info. */
   locator?: string;
+  /** Type specifier for the result of the expression. */
+  resultTypeSpecifier?: AnyTypeSpecifier;
```

Comment:
Hmm... this appears to be a nonstandard ELM field. I think it is okay that you brought the definition in from `cql-execution`. This should be annotated in the context to indicate that this may or may not be included in translator output.